openeo-geotrellis/src/main/scala/org/openeo/geotrellis/layers/OscarsOpenSearch.scala

```diff
@@ -0,0 +1,69 @@
+package org.openeo.geotrellis.layers
+
+import geotrellis.proj4.LatLng
+import geotrellis.vector.{Extent, ProjectedExtent}
+import org.openeo.geotrellis.layers.OpenSearchResponses.{Feature, FeatureCollection}
+import scalaj.http.HttpOptions
+
+import java.net.URL
+import java.time.ZoneOffset.UTC
+import java.time.format.DateTimeFormatter.ISO_INSTANT
+import java.time.{LocalDate, OffsetTime, ZonedDateTime}
+import scala.collection.Map
+
+object OscarsOpenSearch {
+  def apply(endpoint: URL) = new OscarsOpenSearch(endpoint)
+}
+
+class OscarsOpenSearch(endpoint: URL) extends OpenSearch {
+  def getProducts(collectionId: String, from: LocalDate, to: LocalDate, bbox: ProjectedExtent,
+                            correlationId: String = "", attributeValues: Map[String, Any] = Map()): Seq[Feature] = {
+    val endOfDay = OffsetTime.of(23, 59, 59, 999999999, UTC)
```

Comment:
Isn't this implementation the same for Creo? (I could be missing a detail here.)

Reply:
In creo we also have to pass the processingLevel somehow. Not sure what the best approach is to get rid of this.